### DISCUSSION TITLE
Can't get i_idx and j_idx

### DISCUSSION THREAD

**Discussion by HUSTNO1WXY** [zju3dv/LoFTR#97] [General]
When I reproduce training, I found that the i_idx and j_idx produced in coarse_matching if I don't load pretrained weights and let model parameters initialize itself. Why?

**Comment by HUSTNO1WXY** [comment]
This question is equal to that the demo will not find a match when don't use 
`
matcher.load_state_dict(torch.load("weights/indoor_ds.ckpt")['state_dict'])
`

**Comment by zehongs** [comment]
Why do you think it should find a match without a pretrained model?
